x-pack/plugins/apm/public/components/app/Settings/agent_configurations/List/index.tsx

```diff
@@ -245,7 +245,9 @@ export function AgentConfigurationList({
         items={configurations}
         initialSortField="service.name"
         initialSortDirection="asc"
-        initialPageSize={20}
+        initialPageSize={25}
```

Comment:
Instead of hardcoded the values can we please store them in a constant variable? it'll improve future changes much more.